Failed to copy to clipboard or download image

I have this JSON:

```json
{
    "string": "\b"
}
```

It happens when the value is one of or a combination of: `\b`, `\f`.

When I try to copy or donwload an image:
![image](https://user-images.githubusercontent.com/24702416/183264515-ef812c35-019a-41dd-854f-7b96040fd4da.png)
it fails:

```
_app-3f3d54d5d6e3969c.js:1 Error inlining remote css file SecurityError: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules
(anonymous) @ _app-3f3d54d5d6e3969c.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
Promise.then (async)
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
N @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
U @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
Promise.then (async)
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
$ @ 124.9f004a03280b513b.js:1
Z @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
$ @ 124.9f004a03280b513b.js:1
W @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
$ @ 124.9f004a03280b513b.js:1
X @ 124.9f004a03280b513b.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
l @ main-f4ae3437c92c1efc.js:1
(anonymous) @ main-f4ae3437c92c1efc.js:1
P.forEach.e.<computed> @ main-f4ae3437c92c1efc.js:1
L @ 344.c5b9ee010345cb96.js:1
c @ 344.c5b9ee010345cb96.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
Oe @ framework-0471df08d5fed370.js:1
Ve @ framework-0471df08d5fed370.js:1
(anonymous) @ framework-0471df08d5fed370.js:1
Ir @ framework-0471df08d5fed370.js:1
Ur @ framework-0471df08d5fed370.js:1
(anonymous) @ framework-0471df08d5fed370.js:1
is @ framework-0471df08d5fed370.js:1
Le @ framework-0471df08d5fed370.js:1
Wr @ framework-0471df08d5fed370.js:1
Kn @ framework-0471df08d5fed370.js:1
Qn @ framework-0471df08d5fed370.js:1
i @ _app-3f3d54d5d6e3969c.js:1
Show 23 more frames
_app-3f3d54d5d6e3969c.js:1 Error loading remote stylesheet SecurityError: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules
(anonymous) @ _app-3f3d54d5d6e3969c.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
Promise.catch (async)
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
Promise.then (async)
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
N @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
T @ 124.9f004a03280b513b.js:1
U @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
Promise.then (async)
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
$ @ 124.9f004a03280b513b.js:1
Z @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
$ @ 124.9f004a03280b513b.js:1
W @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
(anonymous) @ 124.9f004a03280b513b.js:1
$ @ 124.9f004a03280b513b.js:1
X @ 124.9f004a03280b513b.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
l @ main-f4ae3437c92c1efc.js:1
(anonymous) @ main-f4ae3437c92c1efc.js:1
P.forEach.e.<computed> @ main-f4ae3437c92c1efc.js:1
L @ 344.c5b9ee010345cb96.js:1
c @ 344.c5b9ee010345cb96.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
(anonymous) @ 344.c5b9ee010345cb96.js:1
Oe @ framework-0471df08d5fed370.js:1
Ve @ framework-0471df08d5fed370.js:1
(anonymous) @ framework-0471df08d5fed370.js:1
Ir @ framework-0471df08d5fed370.js:1
Ur @ framework-0471df08d5fed370.js:1
(anonymous) @ framework-0471df08d5fed370.js:1
is @ framework-0471df08d5fed370.js:1
Le @ framework-0471df08d5fed370.js:1
Wr @ framework-0471df08d5fed370.js:1
Kn @ framework-0471df08d5fed370.js:1
Qn @ framework-0471df08d5fed370.js:1
i @ _app-3f3d54d5d6e3969c.js:1
Show 24 more frames
_app-3f3d54d5d6e3969c.js:1 Error while reading CSS rules from https://microsoft.github.io/monaco-editor/node_modules/monaco-editor/min/vs/editor/editor.main.css SecurityError: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules
```

![image](https://user-images.githubusercontent.com/24702416/183264543-23b96290-d145-4e4c-baf7-e3ed6016189b.png)

--- a/src/containers/Editor/LiveEditor/helpers.ts
+++ b/src/containers/Editor/LiveEditor/helpers.ts
@@ -6,7 +6,10 @@ const toString = (value: string | object) => {
 
   if (isObject) {
     const entries = Object.entries(value);
-    const stringObj = entries.map((val) => [val[0], String(val[1])]);
+    const stringObj = entries.map((val) => [
+      JSON.stringify(val[0]).replaceAll('"', ""),
+      JSON.stringify(val[1]),
+    ]);
 
     return Object.fromEntries(stringObj);
   }

